### ISSUE TITLE
Issue with Better Combat + First-person Model 

### ISSUE THREAD

**Issue by Sereliana** [Luke100000/ImmersiveArmors#97]
When playing with the Better Combat and First-person Model mods the helmets show up during attack animations. This only happens with armors and helmets from this mod for some reason, others work fine. Playing on Forge 1.19.2, also tested with only those relevant mods enabled.

Mod Versions:
bettercombat-forge-1.7.1+1.19
cloth-config-8.3.103-forge
firstperson-forge-2.2.3-mc1.19.2
immersive_armors-1.5.6+1.19.2-forge
player-animation-lib-forge-1.0.2

Example during sword stab animation:
![2023-08-25_15 04 47](https://github.com/Luke100000/ImmersiveArmors/assets/72799237/b301a1b3-88a5-4679-a176-f890cd04a1eb)

**Comment by mcd256** [comment]
I've also been having this issue. My guess is that the helmets use special 3d models that First Person Render doesn't know to hide. If you're looking for a band-aid fix you can install Cosmetic Armor Reworked and hide your helmet with it. You won't be able to see the helmet in third person then but you should be able to see when you do those attack animations in first person.

**Comment by johalun** [comment]
Same here on neoforge 26.2. Love this armor mod but this bug is a deal breaker for our new pack which uses better combat :(  Note: other modded armor doesn't have this issue.

**Comment by Luke100000** [comment]
I can not reproduce this on 26.1.2 Neoforge, 1.19.2 forge, nor Fabric 1.21.1 with just immersive armors, better combat, and firstperson model. Can't test in 26.2 since first person model doesnt exist for this version yet.

**Comment by johalun** [comment]
That tracks. I did not see this issue in 1.21.1 neoforge.

**Comment by johalun** [comment]
> first person model doesnt exist for this version yet

Is that a neoforge dependency? I know it's still in beta for 26.2.

**Comment by alexchernogor19-boop** [comment]
Same problem on Fabric, Minecraft version 26.1.2.

**Comment by alexchernogor19-boop** [comment]
How can I remove the first-person model? It really gets in the way.

**Comment by johalun** [comment]
> It really gets in the way.

Yeah if there's a workaround like a custom datapack to disable the model that'd be great.

**Comment by alexchernogor19-boop** [comment]
For now, I’ve installed a mod that locally hides the display of heavy helmets only. Seems to work.  Name Armor Hider

**Comment by johalun** [comment]
<img width="300" height="283" alt="Image" src="https://github.com/user-attachments/assets/f3d3c095-df35-4fd9-8c3d-b3d3d176f1be" />
<img width="280" height="279" alt="Image" src="https://github.com/user-attachments/assets/c0f82f93-d515-465d-8d28-66e34c9ff0f2" />

As can be seen in this screenshot. The leggings are fine. When I wear only leggings, only the leggings render. When I wear any other piece, helmet, chestplate and boots are rendered. Could be related to the helmet render issue when swinging a sword? This rendering issue exists regardless of better combat being loaded or not.

**Comment by Luke100000** [comment]
Make sure to use the most recent version @johalun
